### PR TITLE
New road line output format for curb and lane lines

### PR DESCRIPTION
**Pull request by serdegaa**

### CODE REVIEW

File: msg/RoadPolyline.msg [hunk 1]
```diff
@@ -0,0 +1 @@
+sensr_msgs/RoadPolylinePoint[] points
```
Comment (marco-seoul-robotics) [inline msg/RoadPolyline.msg]: seems like you can use `geometry_msgs/Vector3.msg` instead

**Review by sebastian-seoul-robotics — APPROVED** [review]
Looks good :+1: Could you remove all code related to the previous lane-line/curbside output?

File: msg/RoadPolyline.msg (hunk 1 above)
Comment (marco-seoul-robotics) [reply msg/RoadPolyline.msg]: http://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Vector3.html

Comment (serdegaa) [reply msg/RoadPolyline.msg]: Yes, thanks!

**Comment by marco-seoul-robotics** [comment]
@serdegaa CMakeLists.txt seems to contain undesirable commas. I think we should remove them :) Could you fix this and make a new PR for it?

**Pull request merged**